extrahop.json

```diff
@@ -1,51 +1,173 @@
-            "required": true,
-            "order": 2
+            "order": 5
+        },
+        "ingestion_type": {
+            "description": "Type of data to retrieve and ingest",
+            "data_type": "string",
+            "order": 6,
+            "value_list": [
+                "Detections",
+                "Metrics",
+                "PCAP"
+            ],
+            "default": "Detections"
+        },
+        "json_object": {
+            "description": "JSON object for on poll action",
+            "data_type": "string",
+            "order": 7
+        },
+        "detection_category": {
+            "description": "Category of detections to retrieve. (Leave blank to retrieve detections in any category)",
+            "data_type": "string",
+            "order": 8
+        },
+        "detection_status": {
+            "description": "Status of detections to retrieve. (Comma-delimited; leave blank to retrieve detections of any status",
+            "data_type": "string",
+            "order": 9
+        },
+        "metric_cycle_length": {
+            "description": "Aggregation time interval for retrieved metrics",
+            "data_type": "string",
+            "order": 10,
+            "value_list": [
+                "auto",
+                "30sec",
+                "5min",
+                "1hr"
+            ],
+            "default": "auto"
+        },
+        "object_type": {
+            "description": "Object type associated with the API IDs to poll for metrics",
+            "data_type": "string",
+            "order": 11,
+            "value_list": [
+                "Application",
+                "Device Group",
+                "Device Group Summary",
+                "Device",
+                "Network"
+            ],
+            "default": "Application"
+        },
+        "object_id": {
+            "description": "API IDs for the objects to poll for metrics (Comma-delimited)",
+            "data_type": "string",
+            "order": 12
+        },
+        "metric_category": {
+            "description": "Category of metrics to retrieve. (See REST API Parameters in the ExtraHop Metric Catalog to find categories)",
+            "data_type": "string",
+            "order": 13
+        },
+        "metric_name": {
+            "description": "Names of metrics to retrieve. (Comma-delimited; see REST API Parameters in the ExtraHop Metric Catalog to find metric names)",
+            "data_type": "string",
+            "order": 14
+        },
+        "bpf": {
+            "description": "Berkeley Packet Filter (BPF) syntax for packet retrieval",
+            "data_type": "string",
+            "order": 15
+        },
+        "limit_bytes": {
+            "description": "Maximum number of return bytes for packet retrieval",
+            "data_type": "string",
+            "order": 16
+        },
+        "limit_search_duration": {
+            "description": "Maximum runtime for packet retrieval",
```

Comment:
Is this in seconds?

Reply:
It is in milliseconds by default, we can add other units using unit suffix as well, Added this change